model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmAdapter.java

```diff
@@ -1838,6 +1849,11 @@ private boolean featureAwareIsOrganizationsEnabled(boolean isOrganizationsEnable
         return isOrganizationsEnabled;
     }
 
+    private boolean featureAwareIsAdminPermissionsEnabled(boolean isAdminPermissionsEnabled) {
```

Comment:
Perhaps we can simplify other places using the same approach with something like:

```
private boolean featureEnabled(Feature feature, boolean isVerifiableCredentialsEnabled) {}
```

Not a blocker, I hope you see my point ...

Reply:
Makes sense, I'll consider update it.